arduino/smartcar/smartcar.ino

```diff
@@ -41,17 +41,35 @@ std::vector<char> frameBuffer;
 void setup() {
   Serial.begin(9600);
 #ifdef __SMCE__
-Camera.begin(QQVGA, RGB888, 15);
-  frameBuffer.resize(Camera.width() * Camera.height() * Camera.bytesPerPixel());
   //mqtt.begin("3.138.188.190", 1883, WiFi);
   //mqtt.begin("aerostun.dev", 1883, WiFi);
   mqtt.begin(WiFi); // Will connect to localhost
+  /*if (mqtt.connect("arduino", "public", "public")) {
+    mqtt.subscribe("/smartcar/switchServer", 0);
+    mqtt.onMessage(+[](String& topic, String& message) {
+      if (topic == "/smartcar/switchServer", 0){
+        mqtt.setHost("3.138.188.190", 1883);
+      }
+    });
+  }*/
```

Comment:
This may also no longer be necessary, as the intended functionality was applied in the code below.

Reply:
This is also removed from the code now.